rest-service/manager_rest/rest/resources_v3/secrets.py

```diff
@@ -41,14 +42,39 @@ def put(self, key):
         """
         Create a new secret
         """
-        key, value = self._validate_secret_inputs(key)
+        request_dict = rest_utils.get_json_and_verify_params({
+            'value': {
+                'type': unicode,
+            },
+            'upsert': {
```

Comment:
please consider changing `upsert` with `update_if_exists` or something..

Reply:
Fixed in 9560882